### ISSUE TITLE
Add. 17,217, foll. 57–58 | https://bl.vuexistapps.us/manuscript/950 (Part21)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#684]
### Problems with Manuscript Overview

- No script has been mentioned in the MS overview. Wright also has not mentioned any script, but it should be assumed that it was Unspecified Syriac script. 

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [X] Missing content

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

**Comment by mikey-tang16** [comment]
Should I add a hand note and leave it empty? This question applies to all entries under Add. 17,217 that do not have a hand note.
@wlpotter

**Comment by wlpotter** [comment]
Yes, please add a hand note for all of these that list it as unspecified Syriac

**Comment by mikey-tang16** [comment]
Handnotes added or adjusted accordingly
